b2sdk/account_info/abstract.py

```diff
@@ -310,9 +314,17 @@ def set_auth_data(
         assert self.allowed_is_valid(allowed)
 
         self._set_auth_data(
-            account_id, auth_token, api_url, download_url, recommended_part_size,
-            absolute_minimum_part_size, application_key, realm, s3_api_url, allowed,
-            application_key_id
+            account_id,
+            auth_token,
+            api_url,
+            download_url,
+            recommended_part_size,
+            absolute_minimum_part_size,
+            application_key,
+            realm,
+            s3_api_url,
+            allowed,
+            application_key_id,
```

Comment:
Stuff like doesn't look particularly good and makes you scroll more but it does make for nice diffs. And I think it's slightly easier to scan the list of arguments this way looking for something - your mileage may vary.